webui/react/src/omnibar/tree-extension/trees/index.ts

```diff
@@ -55,8 +56,9 @@ const root: NonLeafNode = {
         const { experiments: exps } = await getExperiments({
           orderBy: 'ORDER_BY_DESC',
           sortBy: 'SORT_BY_END_TIME',
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          states: Array.from(terminalRunStates).map((s) => 'STATE_' + s) as any,
+          states: Array.from(terminalRunStates).map(
+            (s) => 'STATE_' + s,
+          ) as GetExperimentsParams['states'],
```

Comment:
You should not be replacing the `as any` cast with a different cast. The goal here is to eliminate the casting.

Reply:
I think casting here is valid

Reply:
Keita and I discussed this offline and the answer to this one is to unify `CompoundRunState` and `GetExperimentsParams`.

Reply:
After the discussion, I investigated it more. 
`export type CompoundRunState = RunState | JobState;`
```ts
export enum RunState {
  Active = 'ACTIVE',
  Paused = 'PAUSED',
  StoppingCanceled = 'STOPPING_CANCELED',
  Canceled = 'CANCELED',
  StoppingCompleted = 'STOPPING_COMPLETED',
  Completed = 'COMPLETED',
  StoppingError = 'STOPPING_ERROR',
  Errored = 'ERROR',
  Deleted = 'DELETED',
  Deleting = 'DELETING',
  DeleteFailed = 'DELETE_FAILED',
  Unspecified = 'UNSPECIFIED',
  Queued = 'QUEUED',
  Pulling = 'PULLING',
  Starting = 'STARTING',
  Running = 'RUNNING',
}
```

```ts
export enum Determinedjobv1State {
    UNSPECIFIED = <any> 'STATE_UNSPECIFIED',
    QUEUED = <any> 'STATE_QUEUED',
    SCHEDULED = <any> 'STATE_SCHEDULED',
    SCHEDULEDBACKFILLED = <any> 'STATE_SCHEDULED_BACKFILLED'
}
```
GetExperimentsParams's `state` type is
```ts
states?: Array<
    | 'STATE_UNSPECIFIED'
    | 'STATE_ACTIVE'
    | 'STATE_PAUSED'
    | 'STATE_STOPPING_COMPLETED'
    | 'STATE_STOPPING_CANCELED'
    | 'STATE_STOPPING_ERROR'
    | 'STATE_COMPLETED'
    | 'STATE_CANCELED'
    | 'STATE_ERROR'
    | 'STATE_DELETED'
```

We cant just simply do `STATE__${RunState.Val}`

Reply:
This one is trickier than expected. 
I tried several ways but i couldn't figure it out. I think it's fine to use `as` here